test-storybooks/portable-stories-kitchen-sink/nextjs/stories/Navigation.stories.tsx

```diff
@@ -73,9 +75,29 @@ export default {
         query: {
           foo: 'bar',
         },
+        prefetch: () => {
+          console.log('custom prefetch');
+        },
       },
     },
   },
 } as Meta<typeof Component>;
 
-export const Default: StoryObj<typeof Component> = {};
+export const Default: StoryObj<typeof Component> = {
+  play: async ({ canvasElement, step }) => {
+    const canvas = within(canvasElement);
+    const routerMock = useRouterMock();
```

Comment:
I think renaming this to `getRouter` is also better to avoid hook linter problems.